UPGRADE-2.8.md

```diff
@@ -475,6 +475,8 @@ Security
    }
    ```
 
+ * The `AbstractVoter` is deprecated. Upgrade to `Voter` instead.
+
```

Comment:
How that? SF 2.8 is not released, how can it be à BC break to revert the changes made on 2.8?